deltacat/compute/compactor_v2/compaction_session.py

```diff
@@ -685,9 +679,18 @@ def merge_input_provider(index, item):
             f"and rcf source partition_id of {rcf_source_partition_locator.partition_id}."
         )
         rcf_source_partition_locator = compacted_partition.locator
+
+    round_completion_file_s3_url = rcf.write_round_completion_file(
```

Comment:
Can we also include an E2E test case for the compaction suite that covers the RCF read failure scenario?

Reply:
I can add that.